destination_test.go

```diff
@@ -95,8 +95,8 @@ func TestDestination_Configure_Write(t *testing.T) {
 	}}
 
 	want := []string{
-		`{"level":"warn","record":{"position":"cjE=","operation":"create","metadata":{"foo1":"bar1","opencdc.version":"v1"},"key":"cmF3LWtleQ==","payload":{"before":null,"after":"cmF3LXBheWxvYWQ="}},"message":"foo!"}`,
-		`{"level":"warn","record":{"position":"cjI=","operation":"update","metadata":{"foo2":"bar2","opencdc.version":"v1"},"key":{"r2-key":1},"payload":{"before":{"r2-payload":true},"after":{"r2-payload":false}}},"message":"foo!"}`,
```

Comment:
This changed because the opencdc version is now added when the record reaches the destination ([here](https://github.com/ConduitIO/conduit-connector-sdk/blob/6a139bba72eee86391f086c4e12dc593c1c9c853/destination.go#L264)).